src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1449,7 +1449,7 @@ private static void ReportAnalyzerExecutionTime(TextWriter consoleOutput, Analyz
 
                     analyzerTimeColumn = getFormattedTime(executionTime);
                     analyzerPercentageColumn = getFormattedPercentage(percentage);
-                    analyzerNameColumn = getFormattedAnalyzerName("   " + kvp.Key.ToString());
+                    analyzerNameColumn = getFormattedAnalyzerName($"   {kvp.Key} ({string.Join(", ", kvp.Key.SupportedDiagnostics.Select(d => d.Id))})");
```

Comment:
💡 Can you make the list of IDs distinct and ordered?

Reply:
Any concerns doing much LINQ in this code path?

Also the CI is passing. So it looks like this wasn't tested. I'm not aware of how to test this 😕

Reply:
> Any concerns doing much LINQ in this code path

Should be fine

> I'm not aware of how to test this

@mavasani ?

Reply:
> I'm not aware of how to test this

It looks like we have a few tests that expect analyzer execution time such as [`ReportAnalyzerOutput`](https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs#L9143). Perhaps a similar test there?

Reply:
@cston Thanks for the pointer. I think I could just update the same test.